web/traffic/js/tracer_connect.js

```diff
@@ -0,0 +1,6 @@
+
+function open_websocket() {
+    var adr = "ws://" + window.location.host + "/ws-traffic";
```

Comment:
I don't understand. Websocket is available on the same host that you opened the page from, so this code always works.

Reply:
I think it will be more future proof - it would decouple the monitor frontend and allow to even use a local copy of it to connect to various MIM instances in the network.